### PR TITLE
Update installation docs

### PR DESCRIPTION
**Pull request by vuil**
### What this PR does / why we need it

Update installation docs
```
    Add notes
    - regarding installation paths
    - on installation in internet-restricted environments

    Also:
    - add doc about `tanzu config eula show`
```

### Which issue(s) this PR fixes
Fixes #

### Describe testing done for PR

make doc-lint

### Release note
```release-note
None
```

### CODE REVIEW

File: docs/quickstart/install.md [hunk 1]
```diff
@@ -84,17 +78,32 @@ EOF
 sudo yum install tanzu-cli # dnf install can also be used
 ```
 
+### Note on installation paths
+
+Package managers have opinions on locations to which binaries are installed.
+If the location of the CLI (e.g `/opt/homebrew/bin/tanzu` by HomeBrew,
```
Comment (marckhouzam) [inline docs/quickstart/install.md:84]: FYI, although I think it doesn't matter for the doc, `/opt/homebrew` is the prefix for Apple M1 machines.  For intel Mac's it is something under `/usr...`.  That is why documentation often says, e.g. `$(brew --prefix)/bin/tanzu`

File: docs/quickstart/install.md [hunk 2]
```diff
@@ -84,17 +78,32 @@ EOF
...
+`/usr/bin/tanzu` by apt) conflicts an existing CLI that needs to be retained for
```
Comment (marckhouzam) [inline docs/quickstart/install.md:85]:
```suggestion
`/usr/bin/tanzu` by apt) conflicts with an existing CLI that needs to be retained for
```

File: docs/quickstart/install.md [hunk 3]
```diff
@@ -84,17 +78,32 @@ EOF
 sudo yum install tanzu-cli # dnf install can also be used
 ```
 
+### Note on installation paths
```
Comment (marckhouzam) [inline docs/quickstart/install.md:81]: Thanks for this new section 👏

File: docs/quickstart/install.md [hunk 4]
```diff
@@ -109,9 +118,18 @@ above by running the following before any other CLI commands:
 
 ### Installing the Tanzu CLI
 
-You can install the Tanzu CLI in the internet-restricted environments by downloading the Tanzu CLI Binary
-from Github Release and copy it to the internet-restricted environment. Once copied follow the steps
-mentioned [here](#from-the-binary-releases-in-the-github-project) to install the Tanzu CLI.
+You can install the Tanzu CLI in the internet-restricted environments by
```
Comment (marckhouzam) [inline docs/quickstart/install.md:121]:
```suggestion
You can install the Tanzu CLI in internet-restricted environments by
```

File: docs/quickstart/install.md [hunk 5]
```diff
@@ -109,9 +118,18 @@ above by running the following before any other CLI commands:
...
+downloading the Tanzu CLI Binary from Github Release and copy it to the
```
Comment (marckhouzam) [inline docs/quickstart/install.md:122]:
```suggestion
downloading the Tanzu CLI Binary from a Github Release and copy it to the
```

File: docs/quickstart/install.md [hunk 6]
```diff
@@ -109,9 +118,18 @@ above by running the following before any other CLI commands:
...
+internet-restricted environment. Once copied follow the steps mentioned
+[here](#from-the-binary-releases-in-the-github-project) to install the Tanzu
+CLI.
+
+There are also different solutions to enabling the use of Package Managers
+within internet-restricted environments. The advantage of leveraging the
+Package Managers is that upgrades of the CLI itself will be more
+straightforward. Since these solutions are somewhat specific to the Package
+Management system and user's envinronments, please contact your system
```
Comment (marckhouzam) [inline docs/quickstart/install.md:131]:
```suggestion
Management system and user's environments, please contact your system
```

**Review by marckhouzam — APPROVED** [review]
Minor adjustments but LGTM after those.

File: docs/quickstart/install.md (hunk 1 above)
Comment (vuil) [reply docs/quickstart/install.md:84]: Thanks, nice catch, I will update the doc as suggested.

**Comment by vuil** [comment]
Thanks @marckhouzam, again.

**Pull request merged**